docs/extra/k8s-cheatsheet.md

```diff
@@ -0,0 +1,98 @@
+# Kubernetes Cheatsheet
+
+## Session Manager SSH
+
+When using AWS Session Manager since the CLI is very limited you will benefit from using these aliases:
+```bash
+alias kubectl='sudo microk8s.kubectl -n codacy'
```

Comment:
This will collide with the existing (customer-facing) docs because we're including the "-n codacy" in the instructions.

It would be better like this:

```suggestion
alias kubectl='sudo microk8s.kubectl'
```

Reply:
@afsmeira asked this before and in this specific case this is related to our microk8s.
Do you think I should make it explicit or move it to the other repo?

Reply:
OK, if this is specific to MicroK8s, then I think the only command where there will be a collision because we're specifying the namespace twice would be on the second step here:

https://github.com/codacy/chart/blob/microk8s-install-docs/docs/install.md

Notice that the end of the MicroK8s quick start guide refers to these instructions:

https://github.com/codacy/chart/blob/microk8s-install-docs/docs/install.md

Reply:
I changed it to be configured like the rest of the commands. What do you think?